source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -0,0 +1,143 @@
+======================
+``mc support inspect``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support inspect
+
+Description
+-----------
+
+:mc-cmd:`mc support inspect` allows you to collect files, including metadata files, from all backend drives based on a specified path. 
+The command produces a zip file that includes all matching files with their respective *host+drive+path*. 
+
+MinIO Engineering may request this output as part of diagnostics in |subnet|.
+
+Use the :mc-cmd:`mc support inspect` command to obtain raw object contents for analysis and upload it to the MinIO Subscription Network, |subnet|.
+Or use the command to download metadata for objects.
+
+You can read the resulting object using the MinIO debugging tool, described below.
+Independent or third-party use of the output for diagnostics or remediation is done at your own risk.
+
+.. note::
+   
+   :mc-cmd:`mc support inspect` requires a MinIO host from October 2021 or later for full functionality. 
+
+Install additional tools to decrypt or decode downloaded files.
```

Comment:
They kind of aren't, we just link out to it - which might be fine, but instead of these lines lets just say "Use the :ref:`decode tool <minio-xl-meta-decode-tool>` to analyze the downloaded data."